### DISCUSSION TITLE
Chair and vice chair roles

### DISCUSSION THREAD

**Discussion by mnm678** [uptane/community#2] [General]
This is a request for nominations for the chair and vice chair roles as discussed [here](https://github.com/uptane/uptane.github.io/discussions/180) and in the 2/13 Uptane meeting.

Please volunteer before the next Uptane meeting (2 weeks).

**Comment by hexsecs** [comment]
I nominate myself for the vice-chair position.

**Comment by iramcdonald** [comment]
Speaking as an Uptane Steering Committee member, I approve of this proposal for nominations for Uptane Chair and Vice Chair roles.  And I wish to thank those who volunteer to serve our Uptane Community as officers.
